R/dvhydrograph-data.R

```diff
@@ -163,7 +180,7 @@ extendStep <- function(toPlot){
   isStep <- 'type' %in% names(toPlot) && toPlot[['type']] == "s"
   
   if(isStep){
-    toPlot$x <- c(toPlot$x,  tail(toPlot$x, 1) + 90000) #this is 2400, if changing to POSTLT, need to use lubridate
```

Comment:
This change was necessary because the step of each period was extending past its day marker and into the next period, causing the new vertical steps to not line up properly.